tests/test_two_sided_market.py

```diff
@@ -280,13 +315,14 @@ def test_market_trade_bid_partial(
         assert isclose(market.bids[trade.residual.id].price, 15)
         assert market.bids[trade.residual.id].buyer == "A"
 
+    @staticmethod
     def test_market_accept_bid_emits_bid_split_on_partial_bid(
-            self, called, market=TwoSidedMarket(bc=MagicMock(), time_slot=pendulum.now())):
+            called, market=TwoSidedMarket(bc=MagicMock(), time_slot=pendulum.now())):
         market.add_listener(called)
         bid = market.bid(20, 20, "A", "A")
         trade_offer_info = TradeBidOfferInfo(1, 1, 1, 1, 1)
         trade = market.accept_bid(bid, energy=1, trade_offer_info=trade_offer_info)
-        assert all([ev != repr(MarketEvent.BID_DELETED) for c in called.calls for ev in c[0]])
+        assert all((ev != repr(MarketEvent.BID_DELETED) for c in called.calls for ev in c[0]))
```

Comment:
Minor, but I think that not even the second set of brackets is needed here.

Reply:
Thanks! I'll fix this in my other in progress PR